src/ZombieDriver.php

```diff
@@ -236,7 +236,16 @@ public function setRequestHeader($name, $value)
      */
     public function getResponseHeaders()
     {
-        return (array) $this->server->evalJS('browser.window._response.headers', 'json');
+        $js = <<<JS
+        if (true == zombieVersionCompare('4.0.0', '>=')) {
+          stream.end(JSON.stringify(browser.window._response.headers.toObject()));
```

Comment:
> the public API to access the response in Zombie 4 is browser.response

Does it solve AJAX problem, when page does several AJAX requests once it's loaded and `browser.response` contains last AJAX request response instead of main page response? For that reason I was using `browser.window._response` instead of `browser.response` (maybe it existed in Zombie 2.x too).